### DISCUSSION TITLE
【求助】从事绳匠工作需要注意什么？

### DISCUSSION THREAD

**Discussion by Random114** [share121/inter-knot#2395] [其他]
最近工作吃瘪了，就注册了一个绳网账号，想从事绳匠工作，请问一下从事绳匠这个职业都需要注意什么呀？

**Comment by hewanghai123** [comment]
作为一名绳匠，我们要注意我们的绳网等级，绳网等级在别人眼里就是你实力的证明，所以我们在接取委托的时候，要注意接取与我们绳匠等级相符的委托

**Comment by xinyuanyoo** [comment]
不要变成绳斤
